main.tf

```diff
@@ -18,11 +19,37 @@ resource "azurerm_public_ip" "landing_zone_vnet_pip_bastion" {
   sku                 = "Standard"
 }
 
+# Azure Bastion Subnet
+resource "azurerm_subnet" "AzureBastionSubnet" {
```

Comment:
Conventionally snake_case like "bastion_subnet" instead of "AzureBastionSubnet"

Reply:
That subnet can't have any name other than "AzureBastionSubnet", it's a Microsoft requirement.

Reply:
You are right, it can't have a name other than AzureBastionSubnet, but Markus is referring to the Resource Specifier. You can name it whatever, the Bastion Host is not interested in that. So for terraform you can name it bastion_subnet to fit out naming convention